tsl/src/nodes/decompress_chunk/batch_queue_fifo.h

```diff
@@ -14,28 +14,32 @@
 inline static void
 batch_queue_fifo_create(DecompressChunkState *chunk_state)
 {
-	batch_array_init(&chunk_state->batch_array,
+	DecompressContext *dcontext = &chunk_state->decompress_context;
```

Comment:
Why here are you declaring a new variable pointer to `chunk_state->decompress_context` but in the other functions no??